storage/entities/transaction.js

```diff
@@ -653,6 +664,24 @@ class Transaction extends BaseEntity {
 
 		return filters;
 	}
+
+	/**
+	 * Checks if a dapp exists.
+	 *
+	 * @param {Object} params
+	 * @param {string} params.transactionId
+	 * @param {string} params.name
+	 * @param {string} params.link
```

Comment:
The objet is `filters` not the `params`.

Reply:
Method was removed